docs/source/examples/Widget Styling.ipynb

```diff
@@ -169,7 +169,42 @@
     }
    ],
    "source": [
-    "from ipywidgets import HBox, Label, IntSlider\n",
+    "from ipywidgets import IntSlider\n",
+    "\n",
+    "IntSlider(description='A too long description')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can set the length of the description are to match the description, the widget's length is shorter in presence of a description. This because the description is added *inside* of the widget's total length."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "style = {'description_width': 'initial'}\n",
+    "IntSlider(description='A too long description', style=style)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you need more flexibility to layout widgets and captions, you should use a combination with the `Label` widgets arranged in a layout."
```

Comment:
How about:

If you need more flexibility to lay out widgets and descriptions, you can use Label widgets directly.